### PR TITLE
update aitargets

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
Updates all AItargets to a fitting tickrate, some were missing delays completly. Adjusted the paused AI, which was re-creating new minimal AI's each tick to simply call stuff like moveToXYZ, removed those two minimal AI's and just added the 5 lines to the worker logic directly. 

Review please

### CODE REVIEW

**Comment by reggaemuffin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/com/minecolonies/coremod/entity/ai/citizen/student/EntityAIStudy.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Minecolonies/minecolonies/pullRequest?prid=4677411&bid=15649246)

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java [hunk 1]
```diff
@@ -86,9 +85,6 @@ protected IAIState crush()
         {
             return getState();
         }
-        WorkerUtil.faceBlock(getOwnBuilding().getPosition(), worker);
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java:89]: why did you remove faceBlock?

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIStructureMiner.java [hunk 2]
```diff
@@ -123,15 +123,15 @@ public EntityAIStructureMiner(@NotNull final JobMiner job)
...
-          new AITarget(START_WORKING, this::startWorkingAtOwnBuilding),
-          new AITarget(PREPARING, this::prepareForMining),
-          new AITarget(MINER_SEARCHING_LADDER, this::lookForLadder),
-          new AITarget(MINER_WALKING_TO_LADDER, this::goToLadder),
-          new AITarget(MINER_CHECK_MINESHAFT, this::checkMineShaft),
-          new AITarget(MINER_MINING_SHAFT, this::doShaftMining),
-          new AITarget(MINER_BUILDING_SHAFT, this::doShaftBuilding),
-          new AITarget(MINER_MINING_NODE, this::executeNodeMining)
+          new AITarget(IDLE, START_WORKING, 1),
+          new AITarget(START_WORKING, this::startWorkingAtOwnBuilding, 20),
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIStructureMiner.java]: maybe make the 20 all to TICKS_SECOND ?

File: src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java [hunk 3]
```diff
@@ -1516,20 +1506,33 @@ private boolean isStartingPaused()
...
-            pausedHandler = new EntityAIStatePausedHandler(worker, getOwnBuilding());
+            return null;
         }
 
-        pausedHandler.doPause();
-        setDelay(WALK_DELAY);
-        return PAUSED;
+        // Pick random activity.
+        final int percent = worker.getRNG().nextInt(100);
+        if (percent < 8)
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java]: magic numbers

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java (hunk 1 above)
Comment (someaddons) [reply src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java:89]: causes the face to juggle left/right all the time, it only works when set each tick

File: src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java (hunk 3 above)
Comment (someaddons) [reply src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java]: yea I just copied them^^

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java (hunk 1 above)
Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java:89]: but now, does he still face the block?

Comment (someaddons) [reply src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java:89]: probably not looking down to it anymore, but he's got the right direction from the pathing

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java:89]: Quite often when I observed him before adding this he was facing the opposite direction =P

**Comment by reggaemuffin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/com/minecolonies/coremod/entity/ai/citizen/student/EntityAIStudy.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Minecolonies/minecolonies/pullRequest?prid=4677411&bid=15649246)

**Comment by reggaemuffin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/com/minecolonies/coremod/entity/ai/citizen/student/EntityAIStudy.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Minecolonies/minecolonies/pullRequest?prid=4677411&bid=15649246)

**Review by marchermans — APPROVED** [review]

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/4145/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

**Comment by marvin-bitterlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ src/main/java/com/minecolonies/coremod/entity/ai/citizen/student/EntityAIStudy.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Minecolonies/minecolonies/pullRequest?prid=4677411&bid=15649246)

**Review by Raycoms — APPROVED** [review]
